### DISCUSSION TITLE
Is PyG support weighted dataset or not?

### DISCUSSION THREAD

**Discussion by agunghabu** [pyg-team/pytorch_geometric#7318] [Q&A] [answered]
I know this looks like beginner question, yeah I'm still new at PyG. I know DGL is support weighted dataset, but how about PyG. Are they support it? I've already did some research for answer in google and other forums, but I still can't get clear answer for this

**Comment by rusty1s** [comment, marked as answer]
Do you mean weighted graphs? Yes, this is supported. The [GNN Cheatsheet](https://pytorch-geometric.readthedocs.io/en/latest/cheatsheet/gnn_cheatsheet.html) lists all layers that can leverage edge weights/edge features.

**Reply by agunghabu** [reply]
Yeah, that's what I mean, thank you for the answer!! 😊
